cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -267,7 +267,12 @@ public KafkaBrokerConfigurationBuilder withListeners(
         }
 
         // Security protocol and Control Plane Listener are configured everywhere
+
+        // Control Plane listener is added to security protocol on KRaft broker only nodes as well because validated by Kafka
```

Comment:
This sentence doesn't male sense to me, what do you mean "because validated by Kafka"?

Reply:
I meant "because the Kafka codebase validates that the control plane listener is in the security protocol list".
So "validated by kafka" is referring to the first part of the sentence to explain why the control plane listener has to be in the security protocol list. Any better suggestion?

Reply:
this information is used by the controller to know which security protocol to use when connecting to the broker's control plane endpoint

Reply:
so @fvaleri @katheris any suggestions about how to make this sentence clearer?